json-rpc/eth_gasPrice.md

```diff
@@ -0,0 +1,17 @@
+# `Eth_gasPrice`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+Specification | Description 
+---|---
+1 |Eth_gasPrice Must return price per gas in wei as type quantity|
+1.1 |Eth_gasPrice must return initial price per gas if there is no information from past transactions|
+1.11|Eth_gasPrice must set initial price per gas before receiving any transactions|
+1.111|Eth_gasPrice's initial pric per gas May be set to 1 |
```

Comment:
1 wei, not 1, best to define before use - initial price should be defined before it is referred to in the other points